### ISSUE TITLE
Replace remaining XAML-drawn icons with Image Service icons

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#10071]
**Blocked by**: https://devdiv-design.visualstudio.com/D3%20Studio/_workitems/edit/12595
Remove and replace XAML-drawn icons with Image Service, once they are available per this blocking issue.  

Cancel/Download are available today, but are very thin and may not work well in our UI. I believe new icons are being created per issue.
For the ones that already had close matches, they'll be removed as part of https://github.com/nuget/home/issues/9789

## Cancel option?
I did find a possible replacement for the Cancel ("X"), but it's very thin and may be too difficult to see. (particularly on Dark theme). Code change: https://github.com/NuGet/NuGet.Client/commit/06902c0a18d480f962295c3cde602a2733ccbb23. 

@JonDouglas can comment on these samples:
![image](https://user-images.githubusercontent.com/49205731/94374270-17ac4080-00d9-11eb-96e0-0c8f42319a6f.png)
![image](https://user-images.githubusercontent.com/49205731/94374273-19760400-00d9-11eb-8428-548725f3f93d.png)
![image](https://user-images.githubusercontent.com/49205731/94374277-1b3fc780-00d9-11eb-9848-73643fde41a0.png)
![image](https://user-images.githubusercontent.com/49205731/94374278-1c70f480-00d9-11eb-98de-7cb6e6b88c7a.png)

## XAML-drawn icons which don't currently have close matches in ImageService:
Icon | PackageIconMonikers | src\NuGet.Clients\NuGet.PackageManagement.UI\Xamls\
-- | -- | --
![image](https://user-images.githubusercontent.com/49205731/94374239-d7e55900-00d8-11eb-94a4-192f788e9433.png)  | PrefixReservedIndicator | PrefixReservedIndicator.xaml
![image](https://user-images.githubusercontent.com/49205731/94374246-e5024800-00d8-11eb-8a49-b681b222b679.png) | UpdateAvailableIndicator | UpdateAvailableIndicator.xaml
  
![image](https://user-images.githubusercontent.com/49205731/94374251-ea5f9280-00d8-11eb-9dd8-6b8c224d7178.png) | UninstallIndicator | UninstallIndicator.xaml
 
![image](https://user-images.githubusercontent.com/49205731/94374253-ee8bb000-00d8-11eb-96e4-5087b9c4bede.png) | DownloadIndicator | DownloadIndicator.xaml

**Comment by donnie-msft** [comment]
andreinitescu [commented](https://github.com/nuget/home/issues/9789#issuecomment-687562896):
(@JonDouglas any input on removing some of these indicators or changing the Update experience?)

> After more thinking, my conclusion is... There should not be any icons! 
> 
> Usually, status icons are very useful, but in this particular context of the NuGet explorer, which is already a complex screen, I find the status icons over the package icons very distracting and they unnecessarily increase the overall complexity of the screen. 
> 
> For seeing the updates, there's already the "Updates" tab with the nice blue pill-like badge. Unfortunately, when you open the NuGet explorer, it takes an extra click to see what the Updates are, because it's a separate screen and it's not the default one. But let's leave this aside for the moment.
> 
> To see what I mean, compare the following two screenshots of the "Installed" screen, the current look and my suggested new look:
> 
> ### Screenshot of the current look
> 
> ![image](https://user-images.githubusercontent.com/743918/92299189-d94bb780-ef58-11ea-83d8-449126ec07b1.png)
> 
> Beside the confusing status icons, the stacked versions on the right side of every package item in the list is another issue.
> 
> ### My mock-up of the new look
> 
> ![image](https://user-images.githubusercontent.com/743918/92299202-f08aa500-ef58-11ea-867d-46b578adc05c.png)
> 
> Having no complicated overlay status icons and only show the current version numbers makes the "Installed" screen look like exactly what I think it's supposed to be: a **simple and clear list of your installed packages**.
> 
> ### Another option for the new look
> 
> Another option I see for a better "Installed" screen is to add an "Update" button on those package items with an update available:
> 
> ![image](https://user-images.githubusercontent.com/743918/92300091-cb019980-ef60-11ea-8f0a-3792393e6e71.png)
> 
> If having the "Update" button both on the package item in the list and in the package details panel on the right, might feel conflicting/confusing, there could only be only one button visible at a time depending on the selection of the list item:
> - If the package in the list is **not selected**, the list item shows the "Update" button (like in my mock-up above)
> - If the package in the list is **selected**, the list item does not show the "Update" button, because the package details panel on the right has the "Update" button for the selected package.
> 
> All these makes the "Updates" tab unnecessary. The blue badge showing the numbers of available updates would be displayed on the "Installed" tab item.
> 
> Also, in a similar way to how the app updates look on Android and iOS, the list could show first the packages which have updates. 
> 
> Not shown in my mock-up is the "Update" button at the top in order to update all packages. It would only appear if there's at least two updates available. 
> 
> I kept the same stacking of the current and update version numbers on the right when there's an update, but I think there might be a better way to show it than stacking them. Maybe something like just show the version of the update, or show "v12.0.2 ➡ v12.0.3" below the "Update" button.
> 
>
